packages/legend-studio/src/stores/sidebar-state/WorkflowManagerState.ts

```diff
@@ -437,3 +521,53 @@ export class WorkspaceWorkflowsState {
     }
   }
 }
+
+export class WorkspaceWorkflowManagerState extends WorkflowManagerState {
+  constructor(editorStore: EditorStore, sdlcState: EditorSDLCState) {
+    super(editorStore, sdlcState);
+    makeObservable(this, {
+      editorStore: false,
+      sdlcState: false,
+      fetchWorkflowsState: observable,
+      logState: observable,
+      workflowStates: observable,
+      fetchAllWorkflows: flow,
+    });
+  }
+
+  get label(): string {
+    return 'WORKSPACE WORKFLOWS';
+  }
+
+  override get workspace(): Workspace {
+    return this.editorStore.sdlcState.activeWorkspace;
+  }
+}
+
+export class ProjectVersionWorkflowManagerState extends WorkflowManagerState {
+  _version: Version | undefined;
+  constructor(
+    editorStore: EditorStore,
+    sdlcState: EditorSDLCState,
+    version: Version | undefined,
+  ) {
+    super(editorStore, sdlcState);
+    makeObservable(this, {
+      editorStore: false,
+      sdlcState: false,
+      fetchWorkflowsState: observable,
+      logState: observable,
+      workflowStates: observable,
+      fetchAllWorkflows: flow,
+    });
+    this._version = version;
+  }
+
+  get label(): string {
+    return `PROJECT WORKFLOWS ${this.version ? `VERSION ${this.version}` : ''}`;
```

Comment:
Maybe I mistaken, but I thought this is for the header title of the sidebar panel, then we should always keep it as `Workflow Manager`, yeah? If so we can remove this `label()`